code/datums/components/scope.dm

```diff
@@ -105,36 +150,41 @@
 	return target_turf
 
 /**
- * Wrapper for zoom(), so in case we runtime we do not get stuck in a bad state
- *
- * Arguments:
- * * user: The mob we are starting zooming on.
-*/
-/datum/component/scope/proc/start_zooming(mob/user)
-	if(zoom(user))
-		zooming = TRUE
-
-/**
- * We start zooming by hiding the mouse pointer, adding our tracker overlay and starting our processing.
+ * We start zooming by adding our tracker overlay and starting our processing.
  *
  * Arguments:
  * * user: The mob we are starting zooming on.
 */
 /datum/component/scope/proc/zoom(mob/user)
 	if(isnull(user.client))
 		return
-	if(zooming)
+	if(HAS_TRAIT(user, TRAIT_USER_SCOPED))
+		user.balloon_alert(user, "already zoomed!")
 		return
-	user.client.mouse_override_icon = 'icons/effects/mouse_pointers/scope_hide.dmi'
```

Comment:
1. why
2. you didnt delete this dmi file

Reply:
1. The mouse pointer being invisible sucks. The reticle or crosshair doesn't follow its position and stays fixed on the center of the screen, so you may as well accidentally move the cursor out of borders while navigating your view around.
2. I forgor :skull: